uint8 FIT=0
uint8 NEAR=1
uint8 OTHER=2
uint8 REVERSE_FIT=3
uint8 REVERSE_NEAR=4
uint8 REVERSE_OTHER=5

int8 command
